core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -458,6 +477,18 @@ public static boolean defaultUseHttp1Pipelining() {
         return DEFAULT_USE_HTTP1_PIPELINING;
     }
 
+    public static int defaultHttp2InitialWindowSize() {
```

Comment:
How about adding javadoc for the newly added methods?

Reply:
Oops forgot to come back after defining the linked methods, added.